smellyBank/src/smellyBank/Account.java

```diff
@@ -0,0 +1,23 @@
+package smellyBank;
```

Comment:
Make sure that you follow package renaming conventions in [java](https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html) change it to smellybank